selenium_tests/cross_listing/page_objects/cross_listing_main_page.py

```diff
@@ -69,9 +69,11 @@ def delete_cross_listing_pairing(self, data_xlist_map_id):
 
     def is_locator_text_present(self, expected_text):
         """
-        Check to that locator text appears on the confirmation page
+        Check that the locator text appears in the confirmation text box
         """
         try:
+            WebDriverWait(self._driver, 60).until(lambda s: s.find_element(
```

Comment:
Is 60 the minimum that works? Have we tried with 30? In any case, either works, but want to check if we can reduce.